nodejs/private/nodejs_toolchains_repo.bzl

```diff
@@ -107,9 +107,27 @@ resolved_toolchain(name = "resolved_toolchain", visibility = ["//visibility:publ
 
     for [platform, meta] in PLATFORMS.items():
         build_content += """
+# Toolchain resolution is quite complex when it has to choose between both target and exec platform.
```

Comment:
Should this comment be in the generated code like this or outside? Should it be repeated for each platform like it is atm?

Reply:
Where would you like it to put? I can move it outside.